.github/workflows/docker.yml

```diff
@@ -0,0 +1,63 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            rot13maxi/key-convertr
```

Comment:
Change this to whatever docker hub registry or remove it if you only want it uploaded to GHCR